mods/default/functions.lua

```diff
@@ -459,6 +459,23 @@ local function leafdecay_on_timer(pos, def)
 
 	minetest.remove_node(pos)
 	minetest.check_for_falling(pos)
+
+	-- spawn a few particles for the removed node
+	local gravity = tonumber(minetest.settings:get("movement_gravity")) or 9.81
+	gravity = vector.new(0, -gravity, 0)
```

Comment:
Wouldn't that cause settings every time?

Reply:
That would allow for changing settings during runtime, which I'd be fine with. I'm assuming `minetest.settings:get()` isn't an intensive command?

Reply:
It is not a performance concern. However this us unnecessary, it would be better code to cache this outside the function. 'movement_gravity' is highly unlikely to change during a server session.

Reply:
If cached at the start of the file it will be available for all code in the file, that could be useful in future.

Reply:
Done.